ros2/transforms/src/sensor.cpp

```diff
@@ -4,36 +4,38 @@
 class Sensor: public rclcpp::Node{
 private:
     void objectCallback(){
-        // All transforms must be correctly time-stamped!
+        // All transforms must be correctly timestamped!
         pose_.header.stamp = this->get_clock()->now();
         pose_.header.frame_id = "sensor_link";
-        if(count_%2){
+        
+        if(count_%2) {
```

Comment:
```suggestion
        if (count_%2) {
```
Is it possible to add a lint file for future code snippets?

It doesn't seem sustainable to manually check for consistent formatting.